techniques/flash/FLASH12.html

```diff
@@ -10,13 +10,13 @@
       <section class="example">
          <h3>Validating a text field</h3>
          
-            <p>In this example, a sample form is shown with two text fields ('name'   and 'zip code'). Both fields are required. When the form's submit button   is pressed, the values of the text fields will be validated. If a   textfield contains an invalid value, an <code class="obj">_accProps</code> object is   created for the textfield, and its <code class="prop">description</code> property is   set the error message. </p>
+            <p>In this example, a sample form is shown with two text fields ('name'   and 'zip code'). Both fields are required. When the form's submit button   is pressed, the values of the text fields will be validated. If a   textfield contains an invalid value, an <code>_accProps</code> object is   created for the textfield, and its <code>description</code> property is   set the error message. </p>
             <div class="note">
-               <p>Instead of using the accessible description, the error text   can also be added to the accessible name (<code class="prop">_accProps.name</code>),   which is supported by a wider range of assistive technology than the <code class="prop">_accProps.description</code>   property. </p>
+               <p>Instead of using the accessible description, the error text   can also be added to the accessible name (<code>_accProps.name</code>),   which is supported by a wider range of assistive technology than the <code>_accProps.description</code>   property. </p>
```

Comment:
```suggestion
               <p>Instead of using the accessible description, the error text can also be added to the accessible name (<code>_accProps.name</code>), which is supported by a wider range of assistive technology than the <code>_accProps.description</code> property. </p>
```

and could benefit from highlighting